### ISSUE TITLE
Basic analysis breaks with cvmfs-dev3 and prints weird output with self compiled ROOT master

### ISSUE THREAD

**Issue by maxgalli** [KIT-CMS/ntuple_processor#32]
Running a simple analysis after sourcing ```/home/wunsch/workspace/root/build_/bin/root``` (self compiled ROOT master) prints 
```ReadStreamerInfo, class:string, illegal uid=-2```

Running the same analysis after sourcing 
```/cvmfs/sft-nightlies.cern.ch/lcg/views/dev3python3/latest/x86_64-centos7-gcc8-opt/setup.sh```
BREAKS.

**Comment by stwunsch** [comment]
Yep, seen that :/ I think the build of ROOT on CVMFS and the local one interfere somehow ... Well the setup is very hacky ;)

**Comment by stwunsch** [comment]
Also the performance is much worse with that build. I don't know yet why ...
